README.md

```diff
@@ -16,15 +15,27 @@
 }
 ```
 
-5. Add a script to your package.json to lint your code:
+4. Add a script to your package.json to lint your code:
 
 ```
 "scripts": {
-    "lint": "eslint ./src",
+    "lint": "eslint ./",
 },
 ```
 
-6. Now you can manually lint your code by running yarn run lint
+5. The `twig` eslint config uses [eslint-config-compat](https://www.npmjs.com/package/eslint-plugin-compat). This will lint your code for browser compatibility. There are default settings but we recommend adding these settings to your package.json file for Twig Science projects:
+
+```
+"browserslist": [
```

Comment:
Just want to check that this is what we intend on supporting: https://browserslist.dev/?q=bGFzdCAyIHZlcnNpb25zLCBub3QgaWUgPD0gMTEsIGlvc19zYWYgPj0gMTIsIG5vdCBvcF9taW5pIGFsbCwgbm90IGRlYWQ%3D

Reply:
Yeah that makes sense. After conversations with Kati I think we will include IOS>14 as well. So this: [defaults, ios_saf >= 14 and not op_mini all](https://browserslist.dev/?q=ZGVmYXVsdHMsIGlvc19zYWYgPj0gMTQgYW5kIG5vdCBvcF9taW5pIGFsbA%3D%3D)